utils/e2e-tests/bash/tests/try-runtime/on-runtime-upgrade-live.sh

```diff
@@ -8,6 +8,7 @@ LIVE_NETWORK_URL="wss://explorer-rpc-ws.mainnet.stages.humanode.io:443"
   --detailed-log-output \
   --runtime "$RUNTIME_WASM_PATH"  \
   on-runtime-upgrade \
+  --checks=try-state \
```

Comment:
I'm thinking, based on https://github.com/paritytech/substrate/pull/13684, this should be
```suggestion
  --checks=all \
```

Reply:
It won't work as it includes `pre-migrate` and `post-migrate` checks. As an example, `pallet-humanode-session` will try to run `pre-migrate` on running network and check that there are no session identities, but `pallet-humanode-session` migration was done on live network and session identities list is not empty now.

Reply:
Let's have a call regarding this, but it seems like `pallet-humanode-session`s `pre-migrate` and `post-migrate` checks are just broken...

Reply:
https://github.com/humanode-network/humanode/pull/1005